webapi/src/main/scala/org/knora/webapi/messages/util/DateUtil.scala

```diff
@@ -429,11 +430,8 @@ object DateUtil {
    * @param date a [[GregorianCalendar]].
    * @return a Julian Day Number.
    */
-  def convertDateToJulianDayNumber(date: GregorianCalendar): Int = {
-    val conv = new JDateTime
-    conv.loadFrom(date)
-    conv.getJulianDate.getJulianDayNumber
-  }
+  def convertDateToJulianDayNumber(date: GregorianCalendar): Int =
+    date.toInstant().atZone(ZoneId.systemDefault()).toLocalDate().getLong(JulianFields.JULIAN_DAY).toInt
```

Comment:
Just a stylistic note, because my IDE tells me:

You have called Java accessor methods with empty argument clause, when calling the Java method you can actually omit the call parenthesis.

For example `date.toInstant` instead of `date.toInstant()`, same goes for the call to `toLocalDate`.

Reply:
UTC constant added (with capitals), parentheses removed.